packages/core/docs/commercetools/configuration.md

```diff
@@ -9,180 +9,69 @@ Commercetools configuration is located in two places:
-Commercetools API configuration from `middleware.config.js`
-
-```js
-interface ApiConfig {
-  uri: string;
-  authHost: string;
-  projectKey: string;
-  clientId: string;
-  clientSecret: string;
-  scopes: string[];
-}
-```
-
-#### `countries`, `currencies`, `locales`
-
-Array of accepted items
-
-```js
-interface LocaleItem {
-  name: string;
-  label: string;
-}
-```
-
-#### `cookies`
+## Middleware Commercetools configuration
 
-Cookie names for `currency`, `country`, and `locale`
+You can read more about middleware configuration in Vue Storefront [here](../advanced/server-middleware.md#configuration)
 
 ```js
-interface CookiesConfig {
-  currencyCookieName: string;
-  countryCookieName: string;
-  localeCookieName: string;
-}
+// middleware.config.js
+module.exports = {
+  integrations: {
+    ct: {
+      location: '@vue-storefront/commercetools-api/server',
+      configuration: {
+        api: {
+          uri: 'https://<DOMAIN_NAME>.com/<PROJECT_KEY>/graphql',
+          authHost: 'https://auth.sphere.io',
+          projectKey: '<PROJECT_KEY>',
+          clientId: '<CLIENT_ID>',
+          clientSecret: '<CLIENT_SECRET>',
+          scopes: [
+            'manage_products:<PROJECT_KEY>',
+            /* other scope rules */
+          ]
+        },
+        currency: 'USD',
+        country: 'US'
+      }
+    }
+  }
+};
 ```
 
-#### `auth`
-
-Token Authentication object. Later extended with custom extension `tokenExtension`
-
-```js
-interface Auth {
-  onTokenChange?: (token: Token) => void;
-  onTokenRead?: () => string;
-  onTokenRemove?: () => void;
-}
-
-interface Token {
-  access_token: string;
-  expires_at: number;
-  expires_in: number;
-  scope: string;
-  token_type: string;
-  refresh_token: string;
-}
-```
+- `uri` - link to your Commercetools GraphQL API instance.
+- `authHost` - link to Commercetools Authentication Server. It is used to request an access token from commercetools OAuth 2.0 service. To choose the nearest service, please visit [Commercetools hosts list](https://docs.commercetools.com/api/authorization)
+- `projectKey` - name of your Commercetools project, i.e. `my-awesome-vsf-project`
+- `clientId` - unique Commercetools Client ID. Visit [Commercetools documentation](https://docs.commercetools.com/tutorials/getting-started#creating-an-api-client) for more details about creating an API Client
+- `clientSecret` - Commercetools secret API key. Visit [Commercetools documentation](https://docs.commercetools.com/tutorials/getting-started#creating-an-api-client) for more details about creating an API Client
+- `scopes` - The scope constrains the endpoints to which a client has access, and whether a client has read or write access to an endpoint. Visit [Commercetools documentation](https://docs.commercetools.com/api/scopes#top) for more details about Scopes.
 
-#### `acceptLanguage`
+### `acceptLanguage`
 
 Commercetools supports querying localised fields via an array of accepted languages - `acceptLanguage`.
```

Comment:
This sentence is not very much human-readable. I know you can explain it better! :)